### ISSUE TITLE
On save methods, don't get Methods Compiler Errors back to GUI (v2.3.9)

### ISSUE THREAD

**Issue by PeterMoe** [jgfoster/Jade.bak#215]
Hi James,

when i have an Error on saving a Method i don't get the Feedback on GUI in v2.3.9.

![NoFeedbackOnCompilerError01](https://user-images.githubusercontent.com/40350726/59671167-7e943780-91bd-11e9-940d-84eb3f7d8c80.png)

Saving without misspelling in Code is possible.

I did some debugging, the Result from the forwarding to Client is an empty String.
Can't debug all because my debugger don't get to the "Forwarded-Code" and my "know how to do" is to bad for correct understanding.

In your older Version 1.077 there is no problem:
![FeedbackOnCompilerError01](https://user-images.githubusercontent.com/40350726/59671406-e34f9200-91bd-11e9-8363-0947803e5240.png)

Can't use the new Jade version when i must find my mistake on my own. ;-)
Since 2 years i will change from v 1.077 to newest Jade, but some little Bugs likes me and GS24x too much. ;-)
I hope you can help me. :-)

Regards
Peter

**Comment by PeterMoe** [comment]
Hi James,

in Jade v2.3.11 i don't get Feedback from server to client if i want to save a method with errors.

Regards,
Peter

**Comment by PeterMoe** [comment]
Hi James,
ich have tested all version now, last "good" version with correct Compiler Error Feedback was v2.3.4.
Since v2.3.5 the save of a method with error does no compiler error feedback to editor.

Last version with JadeServer.ERROR file is 2.3.6, all higher versions (v2.3.7+) don't generate a JadeServer.ERROR file:

Error Feedback from v2.3.6 (JadeServer.ERROR):
```
CompilerError 1001: compilation errors -- parameter 1 is error descriptor
02.08.2019 23:39:59
1 ComplexBlock in JadeServer >> systemBrowser: @19 line 13  [GsMethod OOP 4247363585]  [receiver OOP 3885357313]
2 ComplexBlock in ExecutableBlock >> ensure: @4 line 11  [GsMethod OOP 10941245441]  [receiver OOP 3561701889]
3 ComplexBlock in ExecutableBlock >> ensure: @6 line 11  [GsMethod OOP 10941245441]  [receiver OOP 3965959681]
4 ComplexVCBlock in JadeServer >> systemBrowser: @26 line 15  [GsMethod OOP 4247363585]  [receiver OOP 4568132353]
5 ExceptionHandler >> doTryBlock: @9 line 7  [GsMethod OOP 10941416193]  [receiver OOP 4568199425]
6 ExceptionHandler >> try:on:do: @15 line 21  [GsMethod OOP 10941334529]  [receiver OOP 4568199425]
7 ComplexVCBlock in ExecutableBlock >> on:do: @2 line 9  [GsMethod OOP 10941246465]  [receiver OOP 4568330497]
8 ComplexVCBlock in JadeServer >> systemBrowser: @29 line 18  [GsMethod OOP 4247363585]  [receiver OOP 4569293313]
9 ExceptionHandler >> caughtExceptionWithAction: @5 line 4  [GsMethod OOP 10941334017]  [receiver OOP 4569299969]
10 ComplexBlock in ExceptionHandler >> caughtEx:number:cat:args:handler: @12 line 13  [GsMethod OOP 10941336577]  [receiver OOP 4560407041]
11 ComplexBlock in ExecutableBlock >> ensure: @4 line 11  [GsMethod OOP 10941245441]  [receiver OOP 4558486785]
12 ComplexBlock in ExecutableBlock >> ensure: @6 line 11  [GsMethod OOP 10941245441]  [receiver OOP 4558487809]
13 ExceptionHandler >> caughtEx:number:cat:args:handler: @16 line 14  [GsMethod OOP 10941336577]  [receiver OOP 4569299969]
14 ComplexBlock in ExceptionHandler >> try:on:do: @10 line 15  [GsMethod OOP 10941334529]  [receiver OOP 4558488833]
15 Exception >> _signal:number:args: @2 line 7  [GsMethod OOP 10940442625]  [receiver OOP 4558499585]
16 System class >> signal:args:signalDictionary: @5 line 13  [GsMethod OOP 3552481281]  [receiver OOP 76033]
17 JadeServer >> sbSaveMethod: @16 line 15  [GsMethod OOP 4558499841]  [receiver OOP 4558507777]
18 JadeServer >> _systemBrowserCommand @124 line 40  [GsMethod OOP 4558512385]  [receiver OOP 4558507777]
19 ComplexBlock in JadeServer >> systemBrowserA: @15 line 10  [GsMethod OOP 4558513409]  [receiver OOP 4558648833]
20 Time class >> secondsElapsedTime: @3 line 11  [GsMethod OOP 10956819457]  [receiver OOP 100865]
21 Time class >> millisecondsElapsedTime: @1 line 6  [GsMethod OOP 10956818689]  [receiver OOP 100865]
22 JadeServer >> _millisecondsElapsedTime: @1 line 3  [GsMethod OOP 4558650113]  [receiver OOP 4558507777]
23 JadeServer >> systemBrowserA: @16 line 4  [GsMethod OOP 4558513409]  [receiver OOP 4558507777]
24 ComplexBlock in JadeServer >> systemBrowser: @4 line 4  [GsMethod OOP 4247363585]  [receiver OOP 4558650369]
25 ExceptionHandler >> doTryBlock: @9 line 7  [GsMethod OOP 10941416193]  [receiver OOP 4569299969]
26 ExceptionHandler >> try:on:do: @15 line 21  [GsMethod OOP 10941334529]  [receiver OOP 4569299969]
27 ComplexBlock in ExecutableBlock >> on:do: @2 line 9  [GsMethod OOP 10941246465]  [receiver OOP 4558655745]
28 JadeServer >> systemBrowser: @31 line 5  [GsMethod OOP 4247363585]  [receiver OOP 3957158657]
29 ComplexBlock in JadeServer64bit >> systemBrowser: @3 line 4  [GsMethod OOP 4558658561]  [receiver OOP 4558662401]
30 ExceptionHandler >> doTryBlock: @9 line 7  [GsMethod OOP 10941416193]  [receiver OOP 4558729217]
31 ExceptionHandler >> try:on:do: @15 line 21  [GsMethod OOP 10941334529]  [receiver OOP 4558729217]
32 ComplexBlock in ExecutableBlock >> on:do: @2 line 9  [GsMethod OOP 10941246465]  [receiver OOP 4558732033]
33 JadeServer64bit >> systemBrowser: @8 line 5  [GsMethod OOP 4558658561]  [receiver OOP 3957158657]
```
I hope you can help.

Regards
Peter

**Comment by PeterMoe** [comment]
hi James,

Now i have found the "correct" issue in your code, you have a misspelling for finding the correct class for error handling, #CompileError must me #Compile **r** Error.

will correct my pull request tomorrow.

Regards,
Peter

**Comment by jgfoster** [comment]
Hi Peter,
Sorry to be slow in responding and leaving you to find your own solution. This looks good and I'll incorporate it once I've reviewed the other issues. (Note that if you submit separate Pull Requests for each issue then it is easier to merge them.)

**Comment by PeterMoe** [comment]
Hi James,

Thanx for your "review". It was fun for me to find my own solution, I love debugging, finding "Bugs" and programming a solution and that all in smalltalk.
I have learned much more now how Jade and Smalltalk Applications (not only gemstone Smalltalk ;-)) works.
Was my first time working with Github, I only worked with my Master, the pull growes on his own.
Sorry, next time i will do it in seperated requests, but I have no idea how it works.(I will study it. ;-)).

Regards,
Peter

P.S.: now it works, thanx.
